app/views/track/atom_feed.atom.erb

```diff
@@ -14,7 +14,8 @@
         content = "<p><strong>Unknown search result type " + result[:model].class.to_s + "</strong></p>"
       end
       # Pull out the heading as separate item, from the partial template
-      content.match(/(<span class="head">\s+<a href="[^>]*">(.*)<\/a>\s+<\/span>)/)
+      content.
+        match(/(<span class="head">\s+<a href="[^>]*">(.*)<\/a>\s+<\/span>)/m)
```

Comment:
this is the simplest fix, not sure if there's any security implication to doing this - the alternative is to add `content.gsub!("\r\n", " ")` to the preceding line instead